samples/opencl/FindOpenCL.cmake

```diff
@@ -0,0 +1,174 @@
+# Find OpenCL
```

Comment:
original file was obtained here: https://github.com/elhigu/cmake-findopencl, no license was found at this place. Added a note to the file about source of original file and modifications that were done in OpenCV version.